CONTRIBUTING.md

```diff
@@ -134,18 +279,25 @@ the test page with your VM. **Do not commit this change**
 Then you should be able bebug using the dev setup: `yarn run dev` and the virtual machine. You can also
 run the page used for function tests using `yarn run test:functional:dev:debug`
 
-## Lint
+## Lint ✨
+
+Lints are static checked for the code. They help us maintain a consistent code base. They are used
```

Comment:
not sure what to replace it with, but "Lints" reads odd

Reply:
We can maybe talk about linters :)

Reply:
Linters are used to statically check the code, they ...